bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2016B/To2016B.scala

```diff
@@ -103,7 +103,13 @@ object To2016B extends Migration {
     }
 
   // Update targets to the new model
-  def updateTargets(d: Document): Unit =
+  def updateTargets(d: Document): Unit = {
+
+    // For normal programs the site is known from the program ID. For weirdo programs like `Andy`
+    // we'll just pick GN arbitrarily. It's the best we can do for now; some observations will end
+    // up with a 1-element ephemeris (which needs to be refreshed anyway) with the wrong site.
+    val site = programSite(d).getOrElse(Ephemeris.empty.site)
```

Comment:
So, nobody picked up on this but after sleeping on it I think it may be a deal-breaker. What would we think if I only did the P2 check for ephemerides with >1 point?